localization/english/wc_schemes_l_english.yml

```diff
@@ -0,0 +1,50 @@
+﻿l_english:
+
+ stasis_trap_title:0 "Stasis Trap:"
+
+ start_stasis_trap:0 "Stasis Trap #weak (Scheme)#!"
+ stasis_trap_scheme:0 "Stasis Trap Scheme"
+ stasis_trap_scheme_desc:0 "[stun|E] your target"
+ stasis_trap_scheme_action:0 "Stasis Trap"
+ stasis_trap_action:0 "Stasis Trap"
+ stasis_trap_desc:0 "I will [stun|E] [SCHEME.GetTarget.GetShortUINameNoTooltip]!"
+ stasis_trap_desc_general:0 "Attempt to [stun|E] an enemy."
+ scheme_interaction_tt_stasis_trap_approved:0 "[recipient.GetFirstName] will soon be [stunned|E]!"
+ stasis_trap_successful_roll_tt:0 "[target.GetShortUINameNoTooltip] becomes [stunned|E]!"
+ stasis_trap_become_discovered_roll_tt:0 "I am discovered!"
+ stasis_trap_scheme_invalidated_title:1 "@scheme_icon! Stasis Trap Ends"
+ stasis_trap_invalidated_i_stunned_message:0 ""
+ stasis_trap_scheme_already_stunned_message:0 "[target.GetShortUIName|U] is already [stunned|E]"
+ stasis_trap_success_discovery_tt:0 "A successful trapping might reveal my identity\nOtherwise:"
```

Comment:
Not sure why but looks like tabs are borken.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117037490-96c0c180-ad17-11eb-9125-ffc8e5512c6a.png)

</details>

Reply:
The abduct one is the same
![bild](https://user-images.githubusercontent.com/4825809/117063012-7e0ad880-ad24-11eb-99b7-41ac8696ffa8.png)

Reply:
You never responded to this one, should it be changed or left as is?

Reply:
> You never responded to this one, should it be changed or left as is?

I see. `stasis_trap_success_discovery_tt` doesn't play any role in this scheme unlike it was in the abduction scheme where it's clear who plotted against you once they finish.

Reply:
Removed in 31e274acfaac6c4d809fee6f92037766b5461951.